api/spec/requests/spree/api/v2/storefront/account/credit_cards_spec.rb

```diff
@@ -107,4 +122,14 @@
       it_behaves_like 'returns 403 HTTP status'
     end
   end
+
+  describe 'credit_cards#destroy' do
+    let!(:credit_card) { create(:credit_card, user_id: user.id) }
+
+    it 'deletes a credit card' do
```

Comment:
@szymoniwacz let's also add a test for deletion attempt of a card that cannot be deleted